modules/virt-importing-vm-cli.adoc

```diff
@@ -169,8 +170,9 @@ EOF
 <9> If `storageMappings` are specified in both the `ResourceMapping` and the `VirtualMachineImport` CRs, the `VirtualMachineImport` CR takes precedence.
 <10> Specify the source storage domain.
 <11> Specify the target storage class.
-<12> Specify the source VM disk ID, for example, `8181ecc1-5db8-4193-9c92-3ddab3be7b05`. You can obtain the disk ID by entering `\https://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` in a web browser on the Manager machine and reviewing the VM details.
-<13> Specify the target storage class.
+<12> Specify `ReadWriteMany` or `ReadWriteOnce`.
```

Comment:
Please add: 
The value of `accessMode` may be one of `ReadWriteOnce`, `ReadWriteMany`, or `ReadOnlyMany`. 
If the access mode is not specified, the oVirt provider will attempt to determine the correct access mode.
based on the VM affinity (Migratable/Non migratable).
Reference:
https://github.com/kubevirt/vm-import-operator/commit/aa71489cf32526a8bc9aa408dd3551b18b56362d#diff-b0929dacd4f1fc192cd308c1a9db2e85154ea742ccec315518e659d7f6a3efc7R103

Reply:
Ilanit, the reference says that if no access mode is specified, the default is `ReadWriteOnce`.

Reply:
If no access mode is specified, the default is ReadWriteOnce - This refers to Import from VMware using VMIO, which we do not document.

Reply:
@pkliczewski  If the source VM migration mode is "Allow manual migration only", what is the volume access mode?